src/applications/caregivers/components/FormFields/FacilitySearch.jsx

```diff
@@ -272,6 +276,10 @@ const FacilitySearch = props => {
           Veteran’s home address.
         </p>
         <va-card role="search" background>
+          <p id="facility-search-description" className="vads-u-margin-top--0">
+            Enter a city, state, or postal code. Then select <b>Search</b> to
```

Comment:
Lets update from the `b` to `strong`. This will provide the context to both sighted and SR users.

Reply:
Done